src/js_schema.hpp

```diff
@@ -389,6 +391,18 @@ ObjectSchema Schema<T>::parse_object_schema(ContextType ctx, ObjectType object_s
         object_schema.is_embedded = false;
     }
 
+    ValueType asymmetric_value = Object::get_property(ctx, object_schema_object, asymmetric_string);
+    if (!Value::is_undefined(ctx, asymmetric_value)) {
+        object_schema.is_asymmetric = Value::validated_to_boolean(ctx, asymmetric_value);
+    }
+    else {
+        object_schema.is_asymmetric = false;
```

Comment:
would it be too much of a change if I turned this into `table_type : Enum`?

Reply:
@danieltabacaru It sounds like a good idea as it will make it impossible to make the error of declaring a table both embedded and asymmetric at the same time.